### DISCUSSION TITLE
About config display option and regex filters

### DISCUSSION THREAD

**Discussion by aborruso** [saulpw/visidata#2478] [General]
Hi,
in my config file I have

```
options.disp_float_fmt = '{:,.02f}'
options.disp_int_fmt = '{:,}'
```

And when I open this

```
{"name": "andy","value":10245888888}
{"name": "saul","value":10245}
```

I have 

![image](https://github.com/user-attachments/assets/99c3b52e-b8c1-4efb-b796-f8f19f6bbd55)

If I run a regex filter to find all rows in value that contain `8888` I have no result.
To have some result I need to apply a right regular expression for what I see, and not for the value contained in the cells.
I must use `8,888`.

I had never noticed this, and I discovered it because I was going crazy trying to figure out why the regex was not working.
Is this the intended behavior? I'm sure it is.

Couldn't this thing become a boolean configuration parameter? And leave it up to the user if he or she wants regexes to apply to what he/she sees and not to cell values. 

And wouldn't it be a good idea to emphasize the actual behavior in the documentation? "Dear user, please note, regex filters are applied to the value as it is formatted."

Best regards,

Andrea

**Comment by aborruso** [comment]
What about this my dear @saulpw ?

**Reply by aborruso** [reply]
@anjakefala Is it better to write this in the issues? Thank you

**Reply by saulpw** [reply]
Probably Issues are better for things that cause you problems, and Discussions are better for longer-form discussions with the community.  I think people check Issues more frequently in general.

**Comment by saulpw** [comment]
Hi @aborruso, yes, you're right, that is how it is intended to work.  We can certainly add a line to that effect to the documentation; where do you think we should put it?  On the command, or on the option you have set, or somewhere else?
